package.json

```diff
@@ -10,6 +10,7 @@
     "@metamask/jazzicon": "https://github.com/jmrossy/jazzicon#7a8df28974b4e81129bfbe3cab76308b889032a6",
     "@rainbow-me/rainbowkit": "^0.11.0",
     "@tanstack/react-query": "^4.24.10",
+    "axios": "^1.4.0",
```

Comment:
Please use the browser native `fetch` API, no need to add a new library just for a POST request :)